data/processed/crunchbase/gbp_venture_capital_received.yaml

```diff
@@ -0,0 +1,29 @@
+api_doc_url: https://github.com/nestauk/data_getters#crunchbase (schema here- https://docs.google.com/presentation/d/1e1aWpQ7M3klHc4iJXJhgpCD77uWFKaFTjVkyvdiNEXw/edit)
+api_type: MySQL
+data_date: 20200218
+description_short: Venture capital investment
+description: Level of venture capital investment in ventures based on a region based on data from CrunchBase. A small number of deals have been converted to GBP at the date when they were announced.
+endpoint_url: https://github.com/nestauk/data_getters#crunchbase
```

Comment:
Can we put the actual endpoint URL here, even if it's not publicly availble?

If it's not, I'd add `is_public` and `auth_provider` to the schema (closing #148 in this PR), and add those 2 values so that it's explicit for the public how to eventually get access, for reproducibility.

Reply:
What do you mean by url endpoint? The S3 location of the database?

What do you mean by `auth_provider`?

Reply:
> The S3 location of the database?
yes

> What do you mean by auth_provider?
See #148